subgraphs/enzyme-core/mappings/v4/AlicePositionLib.ts

```diff
@@ -0,0 +1,41 @@
+import { Address } from '@graphprotocol/graph-ts';
+import { OrderIdAdded, OrderIdRemoved } from '../../generated/contracts/AlicePositionLib4Events';
+import { AliceOrder } from '../../generated/schema';
+import { useAliceOrder, useAlicePosition } from '../../entities/AlicePosition';
+import { ensureAsset } from '../../entities/Asset';
+import { createAssetAmount } from '../../entities/AssetAmount';
+import { toBigDecimal } from '@enzymefinance/subgraph-utils';
+import { useVault } from '../../entities/Vault';
+import { ensureComptroller } from '../../entities/Comptroller';
+
+function handleOrderIdAdded(event: OrderIdAdded): void {
```

Comment:
Through testing I noticed that the OrderIdAdded event did not get picked up by the subgraph. 

Could this be because this function is not exported? Can we fix this and re-deploy?